schema/defs.yaml

```diff
@@ -344,6 +344,15 @@ description: Common schema definitions shared by all themes
           title: cartography
           type: object
           properties:
+            prominence:
+              description: >-
+                Represents Overture's view of a place's significance or importance.
+                This value can be used to help drive cartographic display of a
+                place and is derived from various factors including, but not
+                limited to: population, capital status, place tags, and type.
+              type: integer
+              minimum: 0
```

Comment:
Should the minimum be `1`? At some level a prominence of zero says to me "it is totally irrelevant and shouldn't be there...."

Reply:
Agreed. A `0` value would essentially represent zero significance, which would indicate that the place probably shouldn't exist in the dataset to begin with. A `1` value should be the minimum here.